### PR TITLE
Avatar+badge

### PR DESCRIPTION
**Pull request by vp5h**
Avatars https://deploy-preview-3--avidcss.netlify.app/components/avatar/avatar
Badges https://deploy-preview-3--avidcss.netlify.app/components/badge/badge

### CODE REVIEW

**Review by irfan-nawaz — APPROVED** [review]
there is some commented code.

File: Components/Badge/Badge.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import url(../Avatar/Avatar.css);
```
Comment (ruchiragoswami) [inline Components/Badge/Badge.css]: Use of only small case letters is a good practice in naming files for a website. More details can be searched online regarding 'file naming conventions' for web developers : )

File: Components/Badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                <div class="content-head">
+                    <h2 class="highlight">Profile Avatar</h2>
+                </div>
+                <div class="content">
+                    <div class="example">
+
+                        <div class="badge-container">
+                            <img src="https://pbs.twimg.com/profile_images/1464429641284784135/aD0otj1B_400x400.jpg"
+                                alt="profile-pic" class="avatar lg" />
+                            <div class="badge bg-success"></div>
```
Comment (ruchiragoswami) [inline Components/Badge/badge.html:83]: I am sure these divs are empty for a reason

File: utilsCSS/utilityClasses.css [hunk 3]
```diff
@@ -114,3 +114,22 @@
 .spacing-xxl {
   padding: 4rem !important;
 }
+
+.font-xs {
+  font-size: 0.5rem !important;
```
Comment (ruchiragoswami) [inline utilsCSS/utilityClasses.css]: "Best practices dictate that ' **!important** 'should be used sparingly, if at all, and certainly never just use to get yourself out of a sticky situation with inheritance and the cascade."  Quoting this from a book.

Use of '!importance' is very rare. If we can avoid using this, it is one of the best practices. 

I suggest to go over this "important" code once more and get back to me. I will approve after that.

**Review by ruchiragoswami — COMMENTED** [review]
Great work Pravesh! I have mentioned some comments, please read them and get back to me. I will approve after that :)

File: utilsCSS/utilityClasses.css (hunk 3 above)
Comment (Sudhanshusoni45) [reply utilsCSS/utilityClasses.css]: Yes important should be avoided

File: Components/Avatar/avatar.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    <link rel="stylesheet" href="./Avatar.css" />
+    <script src="https://kit.fontawesome.com/e06444868d.js" crossorigin="anonymous"></script>
+    <title>Avatars | Avid</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="logo">
+            <a href="/index.html">
+                <img class="logo" src="../../Assets/light.png" />
```
Comment (Sudhanshusoni45) [inline Components/Avatar/avatar.html:18]: Give alt to all images for better accessibility

File: Components/Badge/Badge.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+.badge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  font-size: 0.5rem;
+  color: white;
+  font-weight: 900;
+  height: 18px;
+  width: 18px;
```
Comment (Sudhanshusoni45) [inline Components/Badge/Badge.css]: Use rems instead of px for responsiveness

File: utilsCSS/utilityClasses.css (hunk 3 above)
Comment (vp5h) [reply utilsCSS/utilityClasses.css:119]: In general, I agree with you, When we say not use important we are taking about nested classes, But in this particular use case, The thought process is to override the font sizes defined in the classes of elements.

Here's a blog talking about using ! Important in utility classes https://www.cssreactions.com/views/important.html

File: Components/Badge/badge.html (hunk 2 above)
Comment (vp5h) [reply Components/Badge/badge.html:83]: Yeah, The passed Text will appear inside the badge

**Comment by vp5h** [comment]
folks, I think I made all the recommended changes
- removing !important
- file names
- alt for images
have a look

**Review by Sudhanshusoni45 — APPROVED** [review]

**Pull request merged**